### PR TITLE
Account admin

### PR DESCRIPTION
**Pull request by beccagorton-1**
- Added my account pages
- Added error validation for passwords
- Added unhappy paths for signing in and checking email
- Added print button to certificate

### CODE REVIEW

File: app/views/brain-development/quiz-1.html [hunk 1]
```diff
@@ -75,43 +75,49 @@ <h1 class="govuk-heading-l">
           </h1>
 
 
-
```
Comment (agoldstone93) [inline app/views/brain-development/quiz-1.html:78]: I'm curious: why did you remove the nunjucks and replace it with html? Is it because you prefer it? I'm asking because in adding the design history i've been putting in nunjucks because (to me) it's cleaner, but I don't want be making life more difficult for you!

Comment (beccagorton-1) [reply app/views/brain-development/quiz-1.html:78]: I'm not that great with nunjucks, I've always learned best with HTML. I can read & understand NJ but I always get the syntax wrong when I'm trying to add to it or edit it 😝 
Also the javascript I've implemented on the radios I learned to do in HTML so I knew it would work haha

Comment (beccagorton-1) [reply app/views/brain-development/quiz-1.html:78]: Using NJ in the design history is fine! Not making life difficult for me at all!

**Review by agoldstone93 — COMMENTED** [review]
Did you mean to commit the empty ```javascripts/radios.js``` file?

File: app/views/brain-development/quiz-1.html [hunk 2]
```diff
@@ -75,43 +75,49 @@ <h1 class="govuk-heading-l">
...
+                    </div>
+                  </div>
+              
+                </fieldset>
+              </div>
               <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible govuk-!-margin-top-4">
 
               <a class="govuk-button govuk-button--secondary" data-module="govuk-button" href="brain-parts">Previous</a>
-              <a class="govuk-button align-right" data-module="govuk-button" href="quiz-1-right">Submit answer</a>
+              <button class="govuk-button align-right" data-module="govuk-button" type="submit">Submit answer</a>
```
Comment (agoldstone93) [inline app/views/brain-development/quiz-1.html:120]: Just need to change the end tag on this

File: app/views/registration/email-errors/no-access.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+{% block main %}
+
+	<div class="govuk-width-container">
+			<main class="govuk-main-wrapper govuk-!-margin-bottom-8">
+
+				<div class="govuk-grid-row">
+
+					<div class="govuk-grid-column-two-thirds-from-desktop">
+
+                        {% from "govuk/components/input/macro.njk" import govukInput %}
```
Comment (agoldstone93) [inline app/views/registration/email-errors/no-access.html:17]: I dont think lines 17, 18 and 19 are needed but it's no biggie

File: app/views/registration/email-errors/not-received.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+{% block main %}
+
+	<div class="govuk-width-container">
+			<main class="govuk-main-wrapper govuk-!-margin-bottom-8">
+
+				<div class="govuk-grid-row">
+
+					<div class="govuk-grid-column-two-thirds-from-desktop">
+
+                        {% from "govuk/components/input/macro.njk" import govukInput %}
```
Comment (agoldstone93) [inline app/views/registration/email-errors/not-received.html:17]: Same as above

File: app/views/registration/forgot-password.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+{% block main %}
+
+	<div class="govuk-width-container">
+			<main class="govuk-main-wrapper govuk-!-margin-bottom-8">
+
+				<div class="govuk-grid-row">
+
+					<div class="govuk-grid-column-two-thirds-from-desktop">
+
+                        {% from "govuk/components/input/macro.njk" import govukInput %}
```
Comment (agoldstone93) [inline app/views/registration/forgot-password.html:17]: And again

File: app/views/registration/sign-in-errors/account-locked.html [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+{% block main %}
+
+	<div class="govuk-width-container">
+			<main class="govuk-main-wrapper govuk-!-margin-bottom-8">
+
+				<div class="govuk-grid-row">
+
+					<div class="govuk-grid-column-two-thirds-from-desktop">
+
+                        {% from "govuk/components/input/macro.njk" import govukInput %}
```
Comment (agoldstone93) [inline app/views/registration/sign-in-errors/account-locked.html:17]: And again

File: app/views/registration/sign-in-errors/not-received.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+{% block main %}
+
+	<div class="govuk-width-container">
+			<main class="govuk-main-wrapper govuk-!-margin-bottom-8">
+
+				<div class="govuk-grid-row">
+
+					<div class="govuk-grid-column-two-thirds-from-desktop">
+
+                        {% from "govuk/components/input/macro.njk" import govukInput %}
```
Comment (agoldstone93) [inline app/views/registration/sign-in-errors/not-received.html:17]: Oh there's loads of them. Maybe dont worry about it

**Comment by beccagorton-1** [comment]
@agoldstone93 do you know about those conflicts? I don't want to break anything by resolving them, as I don't quite understand what it is that's changed, it's done this a few times before
